OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs

```diff
@@ -367,9 +367,10 @@ public void RefreshServerList()
 						}
 					}
 				}
-				catch
+				catch (Exception e)
 				{
 					searchStatus = SearchStatus.Failed;
+					Log.Write("debug", e.StackTrace);
```

Comment:
Will the exception caught here indicate that it was thrown while the server list was queried, or should we add a line above with `Log.Write("debug", "Server list query failed. Error was:");`?

Reply:
Sure, the stacktrace will mention this file.

Reply:
The stacktrace mentioning this file might not be helpful since it does not necessarily contain the line number or even this function as top-level call.

Reply:
Added it due to popular demand.

Reply:
Thanks